app/controllers/results_controller.rb

```diff
@@ -29,6 +30,17 @@ def index
           address4
           postcode
         ]).all
+    @subjects =
+      if params["subjects"].present?
+        SubjectArea.includes(:subjects).all
+          .map(&:subjects).flatten
+          .select { |subject| rails_to_csharp_subject_id(id: subject.id).in?(params["subjects"].split(",").map(&:to_s)) }
+          .sort_by(&:subject_name)[0..3]
+      else
+        SubjectArea.includes(:subjects).all
+          .map(&:subjects).flatten
+          .sort_by(&:subject_name)[0..3]
+      end
```

Comment:
I may have missed something but I didn't see any tests for this. I was thinking specifically testing that the results are sorting by subject name

Reply:
I didn't add a test because we don't seem to be testing the controllers https://github.com/DFE-Digital/find-teacher-training/tree/master/spec/controllers.
This is tested in the feature test. I should however also test the ordering of the "all subjects" since this is coming from a different part of code.

Reply:
Ah we aren't testing the controllers, I thought I had missed something.
Thanks for adding the tests 👍